storage/migrations/03_agg_stats.up.sql

```diff
@@ -45,6 +45,19 @@ CREATE MATERIALIZED VIEW stats.daily_tx_volume AS
   FROM stats.min5_tx_volume AS sub
   GROUP BY 1, 2;
 
+-- daily_active_accounts stores the sliding widnow for the number of unique accounts per day
+-- that were involved in transactions.
+CREATE TABLE stats.daily_active_accounts
+(
+  layer           TEXT NOT NULL,
+  window_end      TIMESTAMP WITH TIME ZONE NOT NULL,
+  active_accounts uint63 NOT NULL,
+
+  PRIMARY KEY (layer, window_end)
+);
+-- Index for efficient query of the daily samples:
+-- WHERE (layer == 'layer' AND window_end AT TIME ZONE 'UTC')::time = '00:00:00').
+CREATE INDEX ix_stats_daily_active_accounts_window_end_time ON stats.daily_active_accounts (layer, ((window_end AT TIME ZONE 'UTC')::time));
```

Comment:
Does this end up getting used? I think it's better to go with

```suggestion
CREATE INDEX ix_stats_daily_active_accounts_window_end ON stats.daily_active_accounts (layer, window_end);
CREATE INDEX ix_stats_daily_active_accounts_daily_windows ON stats.daily_active_accounts (layer, window_end) WHERE ((window_end AT TIME ZONE 'UTC')::time = '00:00:00'));
```

The first index matches `WHERE` and the `ORDER BY` and will help the db walk the rows in order. Otherwise, it has to do a full table scan and then sort the rows.
The second index is the same, but for the daily query. Compared to the current query: Using `WHERE` instead of indexing the `::time` keeps the index smaller. Also, indexing the `window_end` makes it so the db an just walk the index and obtain sorted rows, instead of using the index to filter on 00:00:00 and then having to sort separately.

Reply:
The first index already exists, due to `(layer, window_end)` being the primary key, that's why I didn't define it explicitly.

The second index did also end up being used for the daily query, but your suggestion is much better indeed. I completely forgot about the option of using a `where` filter in the index.

Reply:
Ah I missed the primary key. Sounds good!